sc-memory/tests/sc-memory/common/test_sc_elements.cpp

```diff
@@ -299,6 +299,66 @@ TEST_F(ScMemoryTest, CreateEdges)
   EXPECT_TRUE(ctx.HelperCheckEdge(linkAddr, nodeAddr, ScType::EdgeUCommonConst));
 }
 
+TEST_F(ScMemoryTest, IterateEraseEdgesBetweenNodes)
+{
+  ScAddr const classAddr = m_ctx->CreateNode(ScType::NodeConstClass);
+  ScAddr const nodeAddr = m_ctx->CreateNode(ScType::NodeConst);
+  
+  size_t const edgesCount = 5;
+  for (size_t i = 0; i < edgesCount; ++i)
+    m_ctx->CreateEdge(ScType::EdgeAccessConstPosPerm, classAddr, nodeAddr);
```

Comment:
```suggestion
    m_ctx->CreateEdge(ScType::EdgeAccessConstPosTemp?, classAddr, nodeAddr);
```

Reply:
Edge types don't influence searching and erasing sc-elements